src/main/java/lotto/domain/generator/LottoGenerator.java

```diff
@@ -0,0 +1,10 @@
+package lotto.domain.generator;
+
+import lotto.domain.number.LottoNumber;
+
+import java.util.HashSet;
+
+public interface LottoGenerator {
+
+    HashSet<LottoNumber> generate();
```

Comment:
```suggestion
    Set<LottoNumber> generate();
```

인터페이스로 구현하면 추후 구현 클래스 변경시 이 메서드를 사용하는 곳에 변화가 발생하지 않는다.